src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/Correspondent.java

```diff
@@ -1,20 +1,114 @@
 package uk.gov.digital.ho.hocs.casework.domain.model;
 
 import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import uk.gov.digital.ho.hocs.casework.application.LogEvent;
 import uk.gov.digital.ho.hocs.casework.domain.exception.ApplicationExceptions;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import javax.persistence.Table;
+import javax.persistence.Transient;
 import java.time.LocalDateTime;
 import java.util.UUID;
 
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 @Entity
 @Table(name = "correspondent")
-public class Correspondent extends BaseCorrespondent {
+@EqualsAndHashCode
```

Comment:
Always suspicious of this on entities, the identity is the UUID, not the value of all the fields.

Reply:
I was thinking that, tempted on just specifying just the UUID.